addons/point_of_sale/static/src/js/models.js

```diff
@@ -3144,6 +3144,10 @@ exports.Order = Backbone.Model.extend({
         } else {
             receipt.footer = this.pos.config.receipt_footer || '';
         }
+        if (!receipt.date.localestring && (!this.state || this.state == 'draft')){
+            receipt.date.localestring = field_utils.format.datetime(
+            moment(new Date()), {}, {timezone: false});
```

Comment:
```suggestion
            receipt.date.localestring = field_utils.format.datetime(moment(new Date()), {}, {timezone: false});
```
Just some nitpick. It would be nice if you consider.